website/docs/patterns/button-alignment/partials/overview/overview.md

```diff
@@ -0,0 +1,98 @@
+While the alignment, order, and grouping of buttons and actions might seem trivial, the impact on users is immediate and often occurs in experiences of high important and interruption.
+
+These guidelines don't cover every scenario and are intended to aid in the decision-making process when determining how to organize buttons and actions. Different methods of alignment and ordering will result in different outcomes with pros and cons for each. What is most important is **consistency** across an application in how actions are aligned, ordered and grouped.
+
+!!! Info
+
+Button alignment isn't relevant for every component or pattern, but is rather aimed at functional elements; i.e. modals, forms, and page-level actions.
```

Comment:
I'm not sure if this is helpful information? Questions of buttons alignment would naturally only apply to cases in which you need to use buttons.